modules/core/src/test/java/org/apache/ignite/internal/lang/IgniteExceptionMapperUtilTest.java

```diff
@@ -204,4 +239,19 @@ public CustomNoMappingException() {
             super(INTERNAL_ERR, "Test internal exception [err=no mapping]");
         }
     }
+
+    /**
+     * Test runtime exception which does not have a mapper.
+     */
+    public static class CustomInternalNotMappedException extends IgniteInternalException {
```

Comment:
We have `CustomNoMappingException` and `CustomInternalNotMappedException` it's confusing a bit.
I suggest use existing `CustomNoMappingException` (just put `code` to the constructor).

Reply:
agree